modules/home/transformations/base/transformations-abstract.php

```diff
@@ -0,0 +1,22 @@
+<?php
+namespace Elementor\Modules\Home\Transformations\Base;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly
+}
+
+abstract class Transformations_Abstract {
```

Comment:
the abstract class does not implement any common functionality 
maybe we can replace the abstract with an interface

Reply:
@TzviRabinovitch I have used it to initialise the json data and the Wordpress adapter in the constructor. Without the abstract class we have to define this data in every transformation separately.

From my perspective the abstract class is more important that the interface.